README.md

```diff
@@ -8,6 +8,15 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
 ---
 
 ## Prelude
+GeekcampSG has been around since 2009. Even though we are (probably) the oldest community conference in Singapore, one of the areas we re lacking, is the knowledge management of the SOPs and good practices for organising the conference. This has resulted in new organisers having to figure out things on their own and sometimes having avoidable blind spots due to their inexperience.
```

Comment:
```suggestion
GeekcampSG has been around since 2009. Even though we are (probably) the oldest community conference in Singapore, one of the areas we are lacking is in the knowledge management of the SOPs and good practices for organising the conference. This has resulted in new organisers having to figure out things on their own, occasionally encountering avoidable blind spots due to their inexperience.
```